packages/daemon/__tests__/utils/wallet.test.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { EventTxOutput } from '../../src/types';
+import { prepareOutputs } from '../../src/utils';
+
+/**
+ * @jest-environment node
+ */
+
+describe('prepareOutputs', () => {
+  it('should ignore NFT outputs', () => {
+    const nftOutputs: EventTxOutput[] = [{
+      value: 1,
+      token_data: 0,
+      script: 'OmlwZnM6Ly9pcGZzL1FtTlJtNmhRUDN2MlVMclVOZTJQTTY4V1dRb2EyUmVwY1IxejVUVVdWZmd0bzGs',
+      // @ts-ignore: This type is wrong, we should allow null here in the type
```

Comment:
suggestion: Replace `ts-ignore` with `ts-expect-error` ( [reference](https://www.totaltypescript.com/concepts/how-to-use-ts-expect-error) ).

By using this annotation you ensure that as soon as this type allows `null`, the test will break and will have to be fixed because there is no longer any error there.

Reply:
Done, thanks!